apis/conversion/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@nebula.js/conversion",
+  "version": "1.0.0",
```

Comment:
I think it's better to set to to version 0.1.0 for now, to indicate it's not complete yet. At least until the basic functionality is in place (like listbox conversion and a nice way to disable object conversion through the Api etc...)

Reply:
Fixed!